backend/lib/services/terminals/index.js

```diff
@@ -13,6 +13,7 @@ const config = require('../../config')
 
 const { Forbidden, UnprocessableEntity } = require('http-errors')
 const { isHttpError } = require('@gardener-dashboard/request')
+const cache = require('../../cache')
```

Comment:
Why do we depend on the cache in terminal bootstrapping. If we move it into an own package we don't have the cache.

Reply:
what speaks against an own (seed) cache for the terminal bootstrapping in the future?

Reply:
removed cache dependency